lexactivator.go

```diff
@@ -824,7 +825,7 @@ func GetLicenseOrganizationAddress(organizationAddress *OrganizationAddress) int
    var cOrganizationAddress = getCArray()
    organizationAddressJson := ""
    status := C.GetLicenseOrganizationAddressInternal(&cOrganizationAddress[0], maxCArrayLength)
-   organizationAddressJson = ctoGoString(&cOrganizationAddress[0])
+   organizationAddressJson = strings.TrimRight(ctoGoString(&cOrganizationAddress[0]), "\x00")
```

Comment:
why was this trimming needed?

Reply:
The json unmarshal failed due to the null character at the end of the string. So had to trim it.
"Error unmarshaling JSON data: invalid character '\x00' looking for beginning of value"